### ISSUE TITLE
Error: Provider produced inconsistent result after apply

### ISSUE THREAD

**Issue by taylorsteiger-rad** [vercel/terraform-provider-vercel#56]
Hi,

When trying to perform a `terraform apply` with a new Vercel Project with `public_source = false`, I got the following error:
<img width="1279" alt="Screen Shot 2022-08-22 at 14 03 00" src="https://user-images.githubusercontent.com/76705912/186018607-aea74347-0fe0-423c-91a2-b5d04634fd01.png">

Below is the resource that was attempted to be created (environment has a list of vars not included):
```
resource "vercel_project" "ecom-storefront" {
  name      = "ecom-storefront-1"
  framework = "nextjs"
  git_repository = {
    type = "github"
    repo = "a-github-repo"
  }
  team_id                    = "a-team-id"
  serverless_function_region = "iad1"
  install_command            = "npm install -g npm-cli-login && npm-cli-login && npm install"
  public_source              = false
  environment = []
}
```

Steps Taken to Debug:

1. Verify `terraform fmt -check`
2. Verify `terraform validate`
3. Verify latest Provider version (using `~> 0.7`)
4. Retry `terraform apply` (destroy works as expected, creation fails again)

Details from `terraform version`:
```
Terraform v1.2.6
on darwin_amd64
+ provider registry.terraform.io/hashicorp/aws v4.27.0
+ provider registry.terraform.io/vercel/vercel v0.7.0
```

**Comment by dglsparsons** [comment]
Hey, thanks for raising this! Looks like a 🐛 on our side. I'll take a look next :)

**Comment by dglsparsons** [comment]
Hey @taylorsteiger-rad, I've just published [v0.7.1](https://github.com/vercel/terraform-provider-vercel/releases/tag/v0.7.1), which should fix this issue. Could you give it a go, and let me know if the issue persists? 

Thanks :)

**Comment by taylorsteiger-rad** [comment]
Thanks, @dglsparsons! That resolved that issue.
